### DISCUSSION TITLE
Is there a way to use Antlers templates in Laravel-only app?

### DISCUSSION THREAD

**Discussion by igorpreston** [statamic/cms#11238] [Q&A] [answered]
I haven't found any straightforward way in guides or docs about this so figured it'd be better to ask directly here. I love Antlers templates and I'm wondering if it's possible to use Antlers as the main template engine (or supplementary) in my main Laravel app without using Statamic CMS there?

For example, if you've heard about Liquid template engine from Shopify - even though it's tightly related to Shopify product - you can use Liquid templates in your own Rails app instead of ERB.

It would be great to use Antlers this way too for those who prefer it or love working with Antlers for some specific use-cases where it's suited much better than Blade.

**Comment by duncanmcclean** [comment, marked as answer]
Unfortunatly, you can't use Antlers without Statamic.

Antlers relies on a lot of Statamic's internal code - it would be hard and likely not that useful for the vast majority of folks to extract it out of Statamic. 

If you're in a Laravel app, I'd recommend just using Blade. It _is_ different, but it's a good rule to follow best practices. (I've inherited Laravel apps with weird templating languages in the past and it wasn't fun 😅)
